iconservice/icon_constant.py

```diff
@@ -115,7 +115,8 @@ class Revision(Enum):
     OPTIMIZE_DIRTY_PREP_UPDATE = 8
 
     FIX_EMAIL_VALIDATION = 9
-    LATEST = 9
+    ADD_LOGS_BLOOM_ON_BASE_TX = 10
```

Comment:
I think you have to check released revision.
because if it doesn't release yet, you should set revision 9 not 10

Reply:
I have checked the release notes and found that the current release revision is 8. I'm going to modify the revision number to 9.

Reply:
Also, I have confirmed this issue to the leader.